### PR TITLE
Add logging to the deployer

### LINKED ISSUE #4: Set up logging
currently not set up for any decent logging -- set up the logging using logstash and/or sentry

### LINKED ISSUE #41: Add possibility to send logs to logstash
Required for testing deployment

### PR DESCRIPTION
**Pull request by rokroskar**
Implements logging in the deployer using the standard python logging module. 

The user can specify the location of a yaml or json logging configuration file using the `RENGA_LOGGING_CONFIG` environment variable. A sample logging config file is included with the deployer, but in general we will have one config file to set up logging for all python-based services. 

closes #4 
closes #41

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
     'requests>=2.18.1',
     'six>=1.10.0',
     'sqlalchemy-utils>=0.32.14',
+    'python-logstash-async>=1.3.1'
```
Comment (jirikuncar) [inline setup.py]: IMHO this should go to `extras_require`

File: renga_deployer/logging.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+from logging import DEBUG, INFO, getLogger
+from logging.config import dictConfig
+
+import yaml
+
+from . import config
+
+conf = config.from_env(config)
+
+if conf.get('RENGA_LOGGING_CONFIG'):
```
Comment (jirikuncar) [inline renga_deployer/logging.py]: we might need to refactor this to a function which is called from `create_app`

File: renga_deployer/contrib/knowledge_graph.py [hunk 3]
```diff
@@ -325,8 +336,8 @@ def mutation(operations, wait_for_response=False, service_access_token=None):
             time.sleep(0.2)
 
         return response
-
-    return response.json()['uuid']
+    else:
```
Comment (jirikuncar) [inline renga_deployer/contrib/knowledge_graph.py]: just `return uuid`

Comment (rokroskar) [reply renga_deployer/contrib/knowledge_graph.py]: fixed in 5a5aabf

**Pull request merged**